tests/fixture/e2e/helpers.go

```diff
@@ -273,26 +288,36 @@ func StartNetwork(
 		// Symlink the path of the created network to the default owner path (e.g. latest_avalanchego-e2e)
 		// to enable easy discovery for reuse.
 		require.NoError(os.Symlink(network.Dir, symlinkPath))
-		tc.Outf("{{green}}Symlinked %s to %s to enable reuse{{/}}\n", network.Dir, symlinkPath)
+		tc.Log().Warn("symlinked network dir for reuse",
```

Comment:
Why is this a `Warn`?
```suggestion
		tc.Log().Info("symlinked network dir for reuse",
```